File: 백준/Bronze/14625. 냉동식품/냉동식품.c
#include <stdio.h>
#include <string.h>

int main() {
    int sH, sM, eH, eM;
    char N[2];
    scanf("%d %d", &sH, &sM);
    scanf("%d %d", &eH, &eM);
    scanf("%s", N);
    int dap = 0;
    while (1) {
        char tmp[5];
        sprintf(tmp, "%02d%02d", sH, sM);  
        for (int i = 0; i < 4; i++) {
            if (tmp[i] == N[0]) {
                dap++;
                break;
            }
        }
        if (sH == eH && sM == eM) {
            break;
        }
        sM += 1;
        if (sM == 60) {
            sH += 1;
            sM = 0;
        }
    }
    printf("%d\n", dap);
    return 0;
}

File: 백준/Bronze/14625. 냉동식품/냉동식품.py
import sys
input = lambda: sys.stdin.readline().rstrip()


def main():
    sH, sM = map(int, input().split())
    eH, eM = map(int, input().split())
    N = input()
    dap = 0
    while 1:
        tmp = str(sH)+str(sM)
        for t in tmp.zfill(4):
            if t == N:
                dap += 1
                break
        if eH == sH and eM == sM:
            break
        sM += 1
        if sM == 60:
            sH += 1
            sM = 0
    print(dap)


if __name__ == '__main__':
    main()